cmd/server.go

```diff
@@ -87,6 +87,7 @@ const (
 	RepoWhitelistFlag          = "repo-whitelist"
 	RepoAllowlistFlag          = "repo-allowlist"
 	RequireApprovalFlag        = "require-approval"
+	RequireUnlockedFlag        = "require-unlocked"
```

Comment:
Maybe let's be specific so it's not confusing for future readers and let's call it: `RequireSQUnlockedFlag`

Reply:
Done!